src/styles/components/_subscribe.scss

```diff
@@ -0,0 +1,83 @@
+@import '../settings/colors';
+@import '../tools/breakpoints';
+
+.w-subscribe {
+  display: block;
+  margin-top: 32px;
+}
+
+.w-subscribe__border {
```

Comment:
If this is only used to give select a border maybe rename to .w-subscribe__select-border